### DISCUSSION TITLE
How do I make a new line (enter) without sending out the prompt?

### DISCUSSION THREAD

**Discussion by shawhu** [oobabooga/text-generation-webui#1838] [General]
right now I can only continue to write in the prompt textbox, how can i make a new line?

**Comment by LaaZa** [comment]
`Shift + Enter`

**Reply by drmbt** [reply]
Installed text-generation-ui today and shift+enter doesn't give me a linebreak, it just submits my prompt... makes it basically unusable. any insight as to why?
